src/assets/svg/social-twitter.svg

```diff
@@ -1 +1 @@
-<svg fill="none" height="20" viewBox="0 0 23 20" width="23" xmlns="http://www.w3.org/2000/svg"><path d="m22 1.01006c-.9576.67547-2.0179 1.19211-3.14 1.53-.6023-.69249-1.4027-1.18331-2.293-1.40608-.8903-.222764-1.8275-.166729-2.6849.16053-.8574.32726-1.5937.90995-2.1091 1.66926-.5155.75932-.7853 1.65862-.773 2.57629v1c-1.75737.04556-3.49873-.34419-5.06899-1.13456-1.57027-.79037-2.92069-1.95681-3.93101-3.39544 0 0-4 9.00004 5 13.00004-2.05947 1.3979-4.51284 2.0989-7 2 9 5 20 0 20-11.50004-.0009-.27855-.0277-.55641-.08-.83 1.0206-1.00651 1.7408-2.27729 2.08-3.67z" stroke="#03c7d3" stroke-linecap="round" stroke-linejoin="round"/></svg>
\ No newline at end of file
+<svg fill="none" height="20" viewBox="0 0 23 20" width="23" xmlns="http://www.w3.org/2000/svg"><path d="m22 1.01006c-.9576.67547-2.0179 1.19211-3.14 1.53-.6023-.69249-1.4027-1.18331-2.293-1.40608-.8903-.222764-1.8275-.166729-2.6849.16053-.8574.32726-1.5937.90995-2.1091 1.66926-.5155.75932-.7853 1.65862-.773 2.57629v1c-1.75737.04556-3.49873-.34419-5.06899-1.13456-1.57027-.79037-2.92069-1.95681-3.93101-3.39544 0 0-4 9.00004 5 13.00004-2.05947 1.3979-4.51284 2.0989-7 2 9 5 20 0 20-11.50004-.0009-.27855-.0277-.55641-.08-.83 1.0206-1.00651 1.7408-2.27729 2.08-3.67z" stroke-linecap="round" stroke-linejoin="round"/></svg>
```

Comment:
There's a Twitter icon in the footer, so I decided to re-use the existing Twitter icon from the Testimonials section. I needed to remove the stroke color, though, for more flexibility.

Reply:
the way i've usually done it in the past is to give the paths a class to indicate their semantic role in the svg, .e.g,

<path class="stdFill">
<path class="primaryStroke">
<path class="secondaryStroke">

So that I can style all the svgs consistently with a consistent scale. i.e., trying to avoid lots of CSS rules like (.svg--github { custom fill custom stroke }) and instead have CSS rules like (.svgHeader .stdFill { color: $color-blue })

The reason we did it this way is that we're trying to optimize for:
- we want to add a new header/footer/menu item, or
- we want to change the styling on all the header/footer/menu-items at once

Reply:
That's a good idea! Will do.